.github/workflows/weekly-featurenets-short-session-tag.yml

```diff
@@ -0,0 +1,91 @@
+---
+name: Weekly test featurenets create (short session, tag)
+
+on:
+  # At 03:40 on Wednesday
+  schedule:
+    - cron: '40 3 * * 3'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  check-vars-and-secrets:
+    name: Check vars and secrets
+    uses: ./.github/workflows/_check-vars-and-secrets.yml
+    secrets: inherit
+
+  get-fqdn-via-tag:
+    name: Get aleph-node main branch FQDN image (short session)
+    needs: [check-vars-and-secrets]
+    runs-on: ubuntu-20.04
+    outputs:
+      fqdn-image: ${{ steps.get-aleph-node-fqdn-image.outputs.fqdn-image }}
+      ref: ${{ steps.get-aleph-node-fqdn-image.outputs.ref }}
+      image-exists: ${{ steps.get-aleph-node-fqdn-image.outputs.image-exists }}
+    steps:
+      - name: Get aleph-node fqdn path
+        id: get-aleph-node-fqdn-image
+        uses: Cardinal-Cryptography/github-actions/get-aleph-node-fqdn-image@v6
+        with:
+          ref: 'r-12.2'
+          test-binary: 'true'
+          ecr-dev-node-repo: ${{ vars.ECR_DEV_ALEPH_NODE_REPO }}
+          ecr-prod-node-repo: ${{ vars.ECR_ALEPH_NODE_REPO }}
+
+  build-and-push-featurenet-node-image-via-tag:
+    needs: [get-fqdn-via-tag]
+    if: ${{ needs.get-fqdn-via-tag.outputs.image-exists != 'true' }}
+    name: Build and push PR test docker image
+    uses: ./.github/workflows/_build-and-push-featurenet-node-image.yml
+    with:
+      ref: ${{ needs.get-fqdn-via-tag.outputs.ref }}
+      fqdn-image: ${{ needs.get-fqdn-via-tag.outputs.fqdn-image }}
+      short-session: true
+    secrets: inherit
+
+  create-featurenet-via-tag:
+    needs:
+      - get-fqdn-via-tag
+      - build-and-push-featurenet-node-image-via-tag
+    # to prevent this job to be skipped when on of the parent jobs is skipped
+    if: ${{ !cancelled() }}
+    name: Create featurenet from tag
+    uses: Cardinal-Cryptography/github-actions/.github/workflows/_featurenet-create.yml@v6
+    secrets: inherit
+    with:
+      featurenet-name: 'ops-test-main-tag'
```

Comment:
I see the name being hardcoded here and there. Any chance of extracting it to some constant at the top of the file and referring to that constant instead? This applies to all files.

Reply:
Actually after refactor it will be better, but it seems there's an [issue](https://github.com/actions/runner/issues/2372) in GHA that prevents to do what [docs](https://docs.github.com/en/github-ae@latest/actions/using-workflows/workflow-syntax-for-github-actions#env) suggests, so I'll skip that one